src/platforms/stlinkv3/platform.c

```diff
@@ -0,0 +1,259 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Copyright (C) 2011-2021 Black Sphere Technologies Ltd.
```

Comment:
The copyright notices for the new files should be updated to `Copyright (C) 2022-2023 1BitSquared  <info@1bitsquared.com>`